### PR TITLE
Permissions clean

### PR DESCRIPTION
**Pull request by flwal-alanda**

### CODE REVIEW

File: frontend/frontend.iml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (mGallee) [inline frontend/frontend.iml:1]: What's the purpose of this xml? Can we remove it?

**Review by mGallee — CHANGES_REQUESTED** [review]

File: frontend/frontend.iml (hunk 1 above)
Comment (flwal-alanda) [reply frontend/frontend.iml]: idea settings, removed them

File: frontend/alanda/libs/common/src/lib/permissions/permissions.directive.ts [hunk 2]
```diff
@@ -29,20 +26,21 @@ interface AlandaPermissionsDirectiveState {
...
 export class AlandaPermissionsDirective {
+
   hostElementManagers: ElementManager[] = getManagersByElementRef(
     this.hostElement,
   );
 
   @Input('alandaPermissions')
   set permission(permissionString: string) {
-    this.rxState.set({ permissionString });
+    this.rxState.set({permissionString});
```
Comment (mGallee) [inline frontend/alanda/libs/common/src/lib/permissions/permissions.directive.ts:36]: This is not properly formatted. Can you run `npm run format:write` to format your code.

**Review by mGallee — APPROVED** [review]

File: frontend/alanda/libs/common/src/lib/permissions/permissions.directive.ts (hunk 2 above)
Comment (mGallee) [inline frontend/alanda/libs/common/src/lib/permissions/permissions.directive.ts:36]: This is not properly formatted. Can you run npm run format:write to format your code.

**Review by mGallee — CHANGES_REQUESTED** [review]

File: frontend/alanda/libs/common/src/lib/permissions/permissions.directive.ts (hunk 2 above)
Comment (flwal-alanda) [reply frontend/alanda/libs/common/src/lib/permissions/permissions.directive.ts]: done

**Review by mGallee — APPROVED** [review]

**Pull request merged**